multiapps-controller-client/src/main/java/org/cloudfoundry/multiapps/controller/client/lib/domain/CloudServiceInstanceExtended.java

```diff
@@ -28,9 +28,19 @@ public boolean isOptional() {
     public boolean isManaged() {
         return false;
     }
-
+    
+    @Value.Default
+    public boolean shouldSkipCredentialsUpdate() {
```

Comment:
"parameters" are the input **given to** the service broker. "credentials" are the things **provided by** the service broker. For example, "memory: 256" would be a parameter, but "host: postgresql.foo.com" would be a credential (see [here](https://apidocs.cloudfoundry.org/13.19.0/service_instances/creating_a_service_instance.html) and [here](https://v3-apidocs.cloudfoundry.org/version/3.89.0/index.html#create-a-service-instance)). This method should be called `shouldSkipParametersUpdate`.